test/integration/smoke/test_deploy_vm_root_resize.py

```diff
@@ -155,6 +155,13 @@ def tearDownClass(cls):
                 Configurations.update(cls.api_client,
                                               "vmware.root.disk.controller",
                                               value=cls.defaultdiskcontroller)
+                StoragePool.update(cls.api_client, id=cls.storageID,
+                                   tags="")
+                cls.restartServer()
+
+                #Giving 30 seconds to management to warm-up,
+                #Experienced failures when trying to deploy a VM exactly when management came up
+                time.sleep(30)
```

Comment:
@borisstoyanov instead of a blind sleep, should we poll every few seconds using wait_until and see if mgmt server's port 8080 is reachable -- see for example, in travis we wait using this: https://github.com/apache/cloudstack/blob/master/tools/travis/before_script.sh#L25

Reply:
This is warmup, we've already used the polling approach to determine management is up, at this stage management is responding, but it need some warmup period.